web/src/components/review.tsx

```diff
@@ -57,8 +59,15 @@ export default function Review() {
   if (languages.length === 0) {
     return (
       <p>
-        You have not selected any languages. Please go to your&nbsp;
-        <Link to="/profile">Profile</Link> to select languages.
+        <Localized
+          id="sc-review-lang-not-selected"
+          elems={{
+            profileLink: <Link to="/profile"></Link>,
+          }}
+        >
+          You have not selected any languages. Please go to your&nbsp;
```

Comment:
Without a `<span>` here the Review component fails when a user has not set up any languages yet. Could you please add it as you have added it to other places such as `language-info.tsx` as well?

Reply:
I must have missed it when doing it for the second time.
Fixed now...